### ISSUE TITLE
Very good work, with a solid theoretical foundation, achieving SOTA results using just a simple quantiser; I would like to ask if there are test codes for object detection and instance segmentation

### ISSUE THREAD

**Issue by Rxx01** [iamkanghyunchoi/mimiq#1]

**Comment by Rxx01** [comment]
The default configuration in the .hocon file is different from the default configuration in the md document, which one should be followed

<img width="1077" height="559" alt="Image" src="https://github.com/user-attachments/assets/f9b90b78-d9bf-46fb-b31a-83a3be6e9aef" />

<img width="771" height="271" alt="Image" src="https://github.com/user-attachments/assets/9b7fc6bc-d36c-42de-b9ce-8814d99f9077" />

**Comment by iamkanghyunchoi** [comment]
Thanks for your interest in our work, and for the thoughtful questions.

* Object detection and instance segmentation test codes

We follow the official MMCV / MMDetection pipelines and use the pretrained weights they provide. After loading the pretrained model, we apply our method to generate synthetic samples and then fine-tune the quantized model. The same quantizer used for the classification experiments is also used for detection and segmentation models.

* Which configuration to follow (.hocon vs. README)

Please refer to the settings described in the README and our paper. In particular, we use the "multi_step" configuration as our default. The .hocon file includes several experimental settings, but the README reflects the configuration used in our main results.

**Comment by Rxx01** [comment]
Thank you so much for your detailed and helpful reply! I wonder if the configuration settings (such as learning rate, optimizer parameters, etc.) for semantic segmentation and object detection tasks are consistent with those for image classification tasks. If it is convenient for you, could you kindly share the corresponding configuration files for reference? It would be greatly appreciated!

**Comment by iamkanghyunchoi** [comment]
Thanks for the follow-up question.

For semantic segmentation and object detection, we use almost the same configuration as in the image classification experiments. The only differences are that we set the learning rate to 1e-4 and use a batch size of 4 due to the larger image sizes. All other optimizer settings and quantization configurations remain unchanged.

**Comment by Rxx01** [comment]
Thank you so much for your clear and detailed clarification! This information is really helpful for my further analysis of the experimental configurations. I appreciate your time and generosity.

**Comment by iamkanghyunchoi** [comment]
Thank you :)
I will close this issue. If anything else comes up, feel free to reopen or create a new one!
